docs/contributors/testing-overview.md

```diff
@@ -448,3 +448,27 @@ To run unit tests only, without the linter, use `npm run test-unit-php` instead.
 
 [snapshot testing]: https://jestjs.io/docs/en/snapshot-testing.html
 [update snapshots]: https://jestjs.io/docs/en/snapshot-testing.html#updating-snapshots
+
+## Performance Testing
+
+To ensure that the editor stays performant as we add feature we monitor the impact PRs and releases can have on some key metrics:
+
+* The time it takes to load the editor.
+* The time it takes for the browser to respond when typing.
+* The time it takes to select a block.
+
+Performance tests are in fact end 2 end tests running the editor and capturing these measures. To run the tests, make sure you have an e2e testing environment ready and run the following command:
```

Comment:
```suggestion
Performance tests are end-to-end tests running the editor and capturing these measures. To run the tests, make sure you have an e2e testing environment ready and run the following command:
```

I think "in fact" can add some value in general (capturing the surprise), but it's a little hard to read in the flow of the sentence.